eq-author-api/schema/resolvers.js

```diff
@@ -191,7 +193,10 @@ const Resolvers = {
   Questionnaire: {
     sections: (questionnaire, args, ctx) =>
       ctx.repositories.Section.findAll({ questionnaireId: questionnaire.id }),
-    createdBy: questionnaire => ({ name: questionnaire.createdBy }),
+    createdBy: questionnaire => ({
+      id: questionnaire.createdBy, // Temporary until next PR introduces users table.
```

Comment:
Can't this use `ctx.auth.name` in this PR?

Reply:
It could I guess, but I'm not sure that makes sense and I don't think it would give you much. The auth token would contain a potentially different name to the one stored against the questionnaire. So you'd end up with something like:

```
createdBy: {
id: "Tom Andrews",
name: "Sam Thomas"
```

The next steps would be to actually store the user in a `users` table. When we get to that stage, we'd resolve the user details from that table and store a foreign key reference on the questionnaire.

Reply:
An `id` field is required for Apollo caching

Reply:
Yes sorry this is wrong. I misread it as the create mutation its the field read. Sorry.